cognite_toolkit/_api/run_api.py

```diff
@@ -0,0 +1,31 @@
+import json
+
+from cognite.client import CogniteClient
+
+from cognite_toolkit._cdf_tk.run import run_function, run_transformation
+from cognite_toolkit._cdf_tk.utils import CDFToolConfig
+
+
+class RunAPI:
+    def __init__(self, cognite_client: CogniteClient):
+        self._client = cognite_client
+
+    def _create_tool_config(self) -> CDFToolConfig:
+        cluster = self._client.config.base_url.removeprefix("https://").split(".", maxsplit=1)[0]
+        cdf_tool_config = CDFToolConfig(cluster=cluster, project=self._client.config.project)
```

Comment:
This will actually work directly in a notebook because a CogniteClient() is instantiated in init if no env vars are available (precedence is CDF_TOKEN env var, then default env vars, then CogniteClient())

Reply:
So I can change it to?
```suggestion
        cdf_tool_config = CDFToolConfig()
```

Reply:
Yes, the vars in CDFToolConig were introduced to set cluster and project from the command line (overriding and env vars). So dropping them will result in both being set from env vars alternatively (in pyiode) instantiating CogniteClient.